base/logging.jl

```diff
@@ -385,9 +385,18 @@ function logmsg_code(_module, file, line, level, message, exs...)
                         end
                         line = $(log_data._line)
                         local msg, kwargs
+                        local trace = if Base.get_bool_env("JULIA_LOG_TRACE", false)
+                            try
+                                error()
+                            catch
+                                stacktrace(catch_backtrace())
+                            end
```

Comment:
Haha, I've literally put this in my code in the past to generate a stacktrace. I wonder if we could have a keyword arg to the log that would also enable it for a single log message.

Reply:
Yeah. I guess it could be put into kwargs without being breaking. For instance
```
local trace = if Base.get_bool_env("JULIA_LOG_TRACE", false) || get(kwargs, :include_trace, false)
```

Reply:
I would maybe change `get(kwargs, :include_trace, false)` to `get(kwargs, :include_trace, false) === true`.

That way, if people currently have code like this:

```
@info "Hello world" include_trace=123
```

Then that code won't suddenly break.

Reply:
added